packaging/win32/makeinstaller.sh

```diff
@@ -16,6 +16,9 @@ iface=${2:-qt5}
 # Note: This script is written to be used with the Fedora mingw environment
 MINGWROOT=/usr/$arch-w64-mingw32/sys-root/mingw
 
+export PKG_CONFIG_ALLOW_CROSS=1
+export PKG_CONFIG_PATH="$MINGWROOT/lib/pkgconfig:$MINGWROOT/"
+
```

Comment:
Curious, why do you need these?

Reply:
I think I saw it in an example of cross-compiled gtk application... indeed it might be not needed... should I check without it and remove?

Reply:
I'ive cross compiled all sorts of stuff and never needed to tweak those, yes please try without

Reply:
Yes, it compiles even without them...